examples/backend/src/ExampleAPI.hs

```diff
@@ -1,10 +1,12 @@
+{-# LANGUAGE BangPatterns #-}
```

Comment:
Where do you need this? I think you can remove it.

Reply:
I used it when using the trace function in a let binding so that it is strictly evaluated, but it was only for testing, so I don't need it now.